air/src/proof/security.rs

```diff
@@ -0,0 +1,728 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+//! Contains helper structs and methods to estimate the security of STARK proofs.
+
+use core::cmp;
+
+use crate::ProofOptions;
+
+const GRINDING_CONTRIBUTION_FLOOR: u32 = 80;
+const MAX_PROXIMITY_PARAMETER: u64 = 1000;
+
+pub struct ConjecturedSecurity(u32);
+
+impl ConjecturedSecurity {
+    pub fn compute(
```

Comment:
Since this is a part of the public interface, let's add doc comments to all public methods of this struct.

Reply:
added